www/src/templates/template-creator-details.js

```diff
@@ -0,0 +1,324 @@
+import presets, { colors } from "../utils/presets"
+import GithubIcon from "react-icons/lib/go/mark-github"
+
+//A variant of the Creators Header Design here with Breadcrumb of Creators > PEOPLE (Whatever) > Creator so clickable to go back to creators
+
+class CreatorTemplate extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { sites: [] }
+  }
+
+  componentDidMount() {
+    this.generateThumbnails()
+  }
+
+  generateThumbnails = () => {
+    let sites = []
+    let creator = this.props.data.creatorsYaml
+    this.props.data.allSitesYaml.edges.map(site => {
+      if (site.node.built_by === creator.name) {
+        sites.push(site)
+      }
+    })
+    this.setState({ sites: sites })
+  }
+  render() {
+    const { data } = this.props
+    const creator = data.creatorsYaml
+    return (
+      <Layout location={location}>
+        <Helmet>
+          <title>{creator.name}</title>
+        </Helmet>
+        <CommunityHeader />
+        <main
+          role="main"
+          css={{
+            padding: rhythm(3 / 4),
+            display: `flex`,
+            justifyContent: `center`,
+            fontFamily: typography.options.headerFontFamily.join(`,`),
+          }}
+        >
+          <div
+            css={{
+              margin: rhythm(3 / 4),
+              flex: `1`,
+              maxWidth: `720`,
+            }}
+          >
+            <Img
+              css={{
+                minWidth: `100%`,
+              }}
+              alt={`${creator.name}`}
+              fluid={creator.image.childImageSharp.fluid}
+            />
+          </div>
+          <div
+            css={{
+              display: `flex`,
+              flexDirection: `column`,
+              margin: rhythm(3 / 4),
+              flex: `1`,
+              maxWidth: `720`,
+            }}
+          >
+            {creator.for_hire || creator.hiring ? (
+              <span css={[styles.badge]}>
+                {creator.for_hire ? `Open For Work` : `Hiring`}
+              </span>
+            ) : null}
+            <span
+              css={{
+                display: `flex`,
+                borderBottom: `2px solid black`,
+                alignItems: `center`,
+              }}
+            >
+              <h1
+                css={{
+                  textTransform: `uppercase`,
+                  margin: `0`,
+                }}
+              >
+                {creator.name}
+              </h1>
+              {creator.github && (
+                <GithubIcon
+                  css={{
+                    marginLeft: `auto`,
+                  }}
+                />
+              )}
+            </span>
+            <span
+              css={{
+                borderBottom: `2px solid black`,
+                padding: `${rhythm()} 0`,
+              }}
+            >
+              <p
+                css={{
+                  margin: `0`,
+                }}
```

Comment:
This is nitpicky, but does it work to combine this `span` and `p`? Unless there's something weird happening, I think combining them would be better for semantics and readability.